RemoteIDModule/BLE_TX.cpp

```diff
@@ -36,8 +36,8 @@ static esp_ble_gap_ext_adv_params_t legacy_adv_params = {
 
 static esp_ble_gap_ext_adv_params_t ext_adv_params_coded = {
     .type = ESP_BLE_GAP_SET_EXT_ADV_PROP_NONCONN_NONSCANNABLE_UNDIRECTED, //set to unicast advertising
-    .interval_min = 1200, //(unsigned int) 0.75*1000/(OUTPUT_RATE_HZ)/0.625; //allow ble controller to have some room for transmission.
-    .interval_max = 1600, //(unsigned int) 1000/(OUTPUT_RATE_HZ)/0.625;
+    .interval_min = 192, //(unsigned int) 0.75*1000/(OUTPUT_RATE_HZ)/0.625; //allow ble controller to have some room for transmission.
+    .interval_max = 267, //(unsigned int) 1000/(OUTPUT_RATE_HZ)/0.625;
```

Comment:
Is there a need to increase the Long Range transmission rate? Wouldn't it be enough to send out those advertisement at an interval slightly less than a second?

Now it sends ~6 of these per second which seems counterintuitive from a power consumption and radio interference viewpoint.